src/catalog/backends/common/src/test/java/org/geoserver/cloud/autoconfigure/catalog/backend/core/RemoteEventResourcePoolCleanupUpAutoConfigurationTest.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * (c) 2024 Open Source Geospatial Foundation - all rights reserved This code is licensed under the
+ * GPL 2.0 license, available at the root application directory.
+ */
+package org.geoserver.cloud.autoconfigure.catalog.backend.core;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.geoserver.cloud.event.info.InfoEvent;
+import org.geoserver.cloud.event.lifecycle.LifecycleEvent;
+import org.geoserver.config.plugin.GeoServerImpl;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.autoconfigure.AutoConfigurations;
+import org.springframework.boot.test.context.FilteredClassLoader;
+import org.springframework.boot.test.context.runner.ApplicationContextRunner;
+
+class RemoteEventResourcePoolCleanupUpAutoConfigurationTest {
```

Comment:
this class should be renamed after the new autoconfiguration class name

Reply:
Done in 33c46d24e75dd4a50af3d908695a1e739d75ce4b (added new tests for both auto configs as both were missing)